### DISCUSSION TITLE
Apply to the GitHub Sponsors program?

### DISCUSSION THREAD

**Discussion by k4b7** [KaTeX/KaTeX#2526] [Ideas]
I was approached by someone who's company was considering sponsoring the project via GitHub Sponsors.  I had a quick look at https://github.com/sponsors and it looks like we have to apply.  I think it would be useful to have some funds on hand to deal with any costs associated with various services we're using.  I know we're trying to keep those costs to zero if possible.

**Comment by edemaine** [comment]
I was looking into this myself (for other projects). The main challenge is that we'd probably need a treasurer, unless KA has something we can plug into. To get a sense, for my personal account, they want a tax form. Then there'd be the issue of routing money to the services... But I agree it's an interesting idea!

**Reply by k4b7** [reply]
I can check with our financial department to see they'd be interested in providing support for the project in that way.

**Reply by ylemkimon** [reply]
@kevinbarabash I wonder what is the status of this. I also got a similar email a few weeks ago.

**Reply by k4b7** [reply]
I've talked to our legal department.  There are a couple of aspects of OSC's terms of service that we need to work through so I'm going to be meeting with them again early in January.  I'm hopeful we'll be able to working out the sticking points and move forward with OSC.

**Reply by k4b7** [reply]
@ylemkimon I've talked with our legal department and have finally gotten around to drafting a proposal which I'll be sending to you, @edemaine, and @ronkok for input.

**Comment by ylemkimon** [comment]
Another way is [Open Source Collective](https://opencollective.com/opensource), which webpack, Jest, Babel, and many other projects use. They handle fiscal matters, support easy expense submission but take 5-10%.
